src/components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../assets/variables.scss";
+
+input[type="checkbox"] {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  -o-appearance: none;
+  appearance: none;
+
+  width: 20px !important;
```

Comment:
Can we achieve that without `!important`?

Reply:
It somehow didn't work for me, but I'll try to change it.